### PR TITLE
Fix release notes for first release in repo

### LINKED ISSUE #2: Release notes fails for first tag in repo because it has no previous release to compare to

### PR DESCRIPTION
**Pull request by nicoleepp**
Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # GitLabChangelog
 [![CI](https://github.com/invenia/gitlabchangelog/workflows/CI/badge.svg)](https://github.com/invenia/gitlabchangelog/actions?query=workflow%3ACI)
-[![Coverage Status](https://coveralls.io/repos/github/invenia/gitlabchangelog/badge.svg?branch=master)](https://coveralls.io/github/invenia/gitlabchangelog)
```
Comment (fchorney) [inline README.md:3]: I guess we're not actually publishing the coverage data and thats why this is removed?

Comment (nicoleepp) [reply README.md:3]: Yes, I tried briefly looking into it but was confused and don't know how to publish it to coveralls

Comment (mattBrzezinski) [reply README.md:3]: I think someone added this in, then didn't add it to Coveralls. Or someone removed it at some point in time as it DNE,

> https://coveralls.io/github/Invenia/
> https://coveralls.io/github/invenia/gitlabchangelog

**Review by fchorney — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nicoleepp) [reply README.md:3]: It's added by cookie cutter but doesn't get set up by cookie cutter

**Pull request merged**